### ISSUE TITLE
chromebox with crouton audio snap, crackle, and pop 

### ISSUE THREAD

**Issue by william555** [dnschneid/crouton#1449]
I have a asus chromebox and installed crouton with sudo sh ~/Downloads/crouton -r trusty -t xfce and then added Kodi, but when I'm playing videos the audio on Kodi is broken up and makes a crackling noise.  Only way I can fix the audio in Kodi is to pause the video, switch back over to Chrome OS, play a video, and move the volume low to high a couple of times, I have to do this every 30 min to an hour.  The crouton and Chrome OS stabe is up to date.  I don't know how to edit the crouton so any instructions would be helpful, thanks.

**Comment by karoliina** [comment]
I have the same issue, except on crouton and XFCE without Kodi. With video files switching to a different player once the noises start seems to work, and VLC is the player that has caused most of the issues. I've changed my default player to SMPlayer now and so far this hasn't happened. The crackling also takes place when watching online streams, in which case I can't just switch players as easily.

**Comment by belzecue** [comment]
Oh thank goodness.  I thought it was just me.  I noticed that creeping audio degradation some weeks ago and just assumed I had munged my Ubuntu/crouton install.  I started wondering if my ASUS chromebox had developed some kind of audio hardware fault, but after testing outside of the chroot, I confirmed ChromeOS itself was unaffected and could play music all day long without the incremental distortion building up as it now does in Crouton/Trusty+XFCE.
